superset-frontend/src/components/Chart/chartAction.js

```diff
@@ -196,9 +198,13 @@ const v1ChartDataRequest = async (
     url,
     headers: { 'Content-Type': 'application/json' },
     body: JSON.stringify(payload),
+    parseMethod: 'text',
   };
 
-  return SupersetClient.post(querySettings);
+  return SupersetClient.post(querySettings).then(({ text, response }) => ({
```

Comment:
@justinpark can we add a simply unit test which includes an integer with a value greater than 2^53 -1?